store/metrics.go

```diff
@@ -86,7 +86,10 @@ const (
 )
 
 func init() {
-	prometheus.MustRegister(readCounter)
+	if prometheus.Register(readCounter) != nil {
+		// Tests will try to double register, so ignore.
```

Comment:
Where else do we register `readCounter`?

Reply:
It's from having tests using both `store_test` and `store` packages; it will init once for `store` and once for `store_test`. I'll clarify the comment

Reply:
There were also problems from v2v3 using the vendored store path and the tests using the unvendored path.